chore: ungate multi-column keys from the feature flag (#6752)
confluentinc/ksql: docs/concepts/schemas.md

A column is defined by a combination of its [name](#valid-identifiers), its [SQL data type](#sql-data-types),
and possibly a namespace.

> Key columns have a `KEY` or `PRIMARY KEY` suffix for streams and tables, respectively while value 

Reviewer: ```suggestion
Key columns for a stream have a `KEY` suffix. Key columns for a table have a `PRIMARY KEY` suffix. Value 

```

> columns have no namespace suffix. There can be multiple columns in either namespace, given that

Reviewer: ```suggestion
columns have no namespace suffix. There can be multiple columns in either namespace, if

```

> the underlying serialization format supports it (the `KAFKA` format, for example, does not).

Reviewer: ```suggestion
the underlying serialization format supports it.

!!! note 
    The `KAFKA` format doesn't support multi-column keys.

```

> For example, the following example statement declares a stream with several

Reviewer: ```suggestion
For example, the following statement declares a stream with multiple

```

substitute `PROTOBUF`, `JSON` or `JSON_SR` for `AVRO` in each statement.

!!! note
    ksqlDB handles the `JSON` and `JSON_SR` formats differently. `JSON_SR` reads and 

>     registers new schemas with {{ site.sr }}, as necessary while `JSON` requires you

Reviewer 2: ```suggestion
    registers new schemas with {{ site.sr }} as necessary, while `JSON` requires you
```
